packages/website/sidebars.ts

```diff
@@ -55,6 +55,17 @@ const sidebars: SidebarsConfig = {
       items: [
         "getting-started/getting-started",
         "getting-started/getting-started-http",
+        "getting-started/getting-started-http-01-setup",
```

Comment:
do you think its maybe better to move this into a sub folder to leave space for other guides?

Reply:
That's fine with me, how does that impact navigation on the website?

Reply:
would show up as a folder on the website as well, you then use 
```ts
{
              "type": "autogenerated",
              "dirName": "getting-started/getting-started-http"
            }
```
to let docusaurus create that list automatically for this folder

Reply:
Okay I made this change based on the similar structure done for 'standard-library', hopefully I did it right.